### ISSUE TITLE
I Get Jupyter notebook 5.7.6 blank page when start "object_detection_tutorial.ipynb"

### ISSUE THREAD

**Issue by Arthon83** [EdjeElectronics/TensorFlow-Object-Detection-API-Tutorial-Train-Multiple-Objects-Windows-10#250]
<script src="/static/components/es6-promise/promise.min.js?v=f004a16cb856e0ff11781d01ec5ca8fe" type="text/javascript" charset="utf-8"></script>
    <script src="/static/components/preact/index.js?v=00a2fac73c670ce39ac53d26640eb542" type="text/javascript"></script>
    <script src="/static/components/proptypes/index.js?v=c40890eb04df9811fcc4d47e53a29604" type="text/javascript"></script>
    <script src="/static/components/preact-compat/index.js?v=aea8f6660e54b18ace8d84a9b9654c1c" type="text/javascript"></script>
    <script src="/static/components/requirejs/require.js?v=951f856e81496aaeec2e71a1c2c0d51f" type="text/javascript" charset="utf-8"></script>

Files could not load.

**Comment by Gain22916** [comment]
I agree with you, Now I try to run Jupyter notebook that is blank page when started "object_detection_tutorial.ipynb".

**Comment by mlnhnk** [comment]
Uninstall notebook 5.7.6 and install version 5.7.5.
Use:
`pip uninstall notebook
pip install notebook==5.7.5`

Everything should work fine then!

**Comment by Arthon83** [comment]
Thanks.

If I have downgrade tornado 5.1.1 it work for me. :)

`pip uninstall tornado`
`pip install tornado==5.1.1`

**Comment by Gain22916** [comment]
Thank you very much , Now I can do it.

**Comment by akshat1145** [comment]
I have the same problem too. Tried the solutions above but could not solve it. Any leads for the same?
